maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -33115,7 +33115,6 @@
 	},
 /obj/item/weapon/circuitboard/clonescanner,
 /obj/item/weapon/circuitboard/clonepod,
-/obj/item/weapon/circuitboard/scan_consolenew,
```

Comment:
Isn't this needed for cloning?

Reply:
Doesn't seem so. `/clonescanner`, `/clonepod` and `/cloning` seem to be boards related to cloning. `/scan_consolenew` is labeled as DNA Machine.

Reply:
It is!

Reply:
Actually, no it isn't!

DNA machinery is fucking retarded in that the machine itself is called `dna_scannernew`, but the circuit inside of it is called `clonescanner`. Like, _What_.